install/kubernetes/helm/istio/templates/crds.yaml

```diff
@@ -1,4 +1,5 @@
-# {{ if or .Values.global.crds (semverCompare ">=2.10.0-0" .Capabilities.TillerVersion.SemVer) }}
+# these CRDs only make sense when pilot is enabled
+# {{- if .Values.pilot.enabled }}
```

Comment:
Does it work ? Usually the values under global are visible, not not each section. 

I'm not sure this is the right solution - pilot is an implementation, so is mixer. The CRDs define the API. 
It is perfectly possible to install Istio without pilot ( maybe pilot is running in a different cluster, or user has 
a different, customized implementation ).

Same for mixer, security - the API definition should be available even if user chooses to not install some components.
They may have apps defining CRs of those type - which are ignored if mixer/security are disabled, but should still
install ( and if they later decide to enable security - the CRs will become active )

Reply:
It does work. :). You can install the whole CRDS.yaml since these are comments.  However having these allow me to not install any CRDs when I don't need them like in my case for gateways.

BTW, what is the case when user install istio without pilot?